app/inpututils.cpp

```diff
@@ -1170,8 +1170,11 @@ qreal InputUtils::calculateScreenDpr()
 
     qreal realDpi = dpiX < dpiY ? dpiX : dpiY;
     realDpi = realDpi * screen->devicePixelRatio();
-
+#ifdef ANDROID // due to crazy values onphysicalDotsPerInch[XY] from Qt6.5 (https://bugreports.qt.io/browse/QTBUG-113751)
```

Comment:
Can you if-def the workaround for QT6.5.0 only? (so when we compile for other qt version  we do not have the workaround)

https://bugreports.qt.io/browse/QTBUG-112742 suggest the value is now in pixels and not in inches, so qt 6.5.0 is basically unusable for us for release with this bug

Reply:
How you come up with the number 8?

Reply:
Added ifdef for Qt6.5.0, will add unfortunately also Qt6.5.1

Reply:
> How you come up with the number 8?

Just by test. Why is there the number 160?